mail_edit/models/compose_message.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+# © 2016 Sunflower IT (http://sunflowerweb.nl)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, fields, models
+
+
+class MailEditComposeMessage(models.TransientModel):
+    _name = "mail_edit.compose_message"
+    _inherits = {"mail.compose.message": "original_wizard_id"}
+
+    @api.model
+    def default_get(self, fields):
+        result = self.original_wizard_id.default_get(fields)
```

Comment:
edit: I think you should call super here, you inherit the defaults. Given that the rest of the code more or less works around not calling super, could you try just getting rid of the function entirely?